src/solver/BCRoutines.F90

```diff
@@ -1240,35 +1240,37 @@ subroutine bcEulerWall(nn, secondHalo, correctForK)
 
     ! Determine the state in the halo cell. Again loop over
     ! the cell range for this subface.
-
```

Comment:
functionality-wise, there should not be any changes in the math. we just changed the flattened loop to a regular double loop. You can keep either version. Mads's version below is better for readability but likely the resulting tapenade code is slower. You can go back to the original version of the code here but you should make sure nothing changes in the result

Reply:
Do I need to run tapenade to see the effect of this change or can I just rebuild ADflow and run my tests?